pkg/webhooks/pod_webhook.go

```diff
@@ -242,10 +242,6 @@ func exclusiveAffinityApplied(pod corev1.Pod, topologyKey string) bool {
 	return hasAffinity && hasAntiAffinity
 }
 
-func getSubGroupIndex(podCount int, subGroupSize int, workerIndex int) string {
-	if (podCount-1)%subGroupSize == 0 {
```

Comment:
It's a new feature called LeaderExcluded just incase the leader is a proxy, should not be part of the subgroup, see.
https://github.com/kubernetes-sigs/lws/blob/a2ae101b56d498c193074ee0b45b07439d62bf6b/api/leaderworkerset/v1/leaderworkerset_types.go#L224

Reply:
Wait, let me revisit the design.

Reply:
https://github.com/kubernetes-sigs/lws/issues/533#issuecomment-2899802572

Reply:
thank you !